packages/mobile/package.json

```diff
@@ -28,7 +28,7 @@
     "ios:device:stage": "ENVFILE=.env.stage turbo run ios -- --scheme 'Staging' --mode Staging.Debug --device",
     "ios:prod": "ENVFILE=.env.prod turbo run ios -- --scheme 'AudiusReactNative' --simulator \"iPhone 15 Pro\"",
     "ios:release": "turbo run ios -- --simulator \"iPhone 15 Pro\" --mode Release",
-    "ios:stage": "ENVFILE=.env.stage turbo run ios -- --scheme 'Staging' --mode Staging.Debug --simulator \"iPhone 15 Pro\"",
```

Comment:
lets undo this?

Reply:
and maybe add a "small device" script if we feel the need?

Reply:
good idea